app/Hydrators/SprintHydrator.php

```diff
@@ -13,6 +13,8 @@ public function hydrate(array $data): Sprint
             $data['id'],
             $data['name'],
             $data['board_id'],
+            new Carbon($data['start_date']),
+            new Carbon($data['end_date']),
```

Comment:
I wonder if we need to see if `start_date` and `end_date` are in the array, since they're nullable.  Something like how we do `updated_at` below.

Reply:
I also kinda wonder which of these is better:
```php
!empty($data['updated_at']) ? new Carbon($data['updated_at']) : null
```
or
```php
array_key_exists('updated_at', $data) ? new Carbon($data['updated_at']) : null
```

Reply:
I just wonder if `empty($data['updated_at'])` will throw a warning if the key doesn't exist

Reply:
Just did some research and it seems fine to use `empty` so lets keep with that, but should think about if `start_date` and `end_date` need that check too

Reply:
Definitely right on this, carbon seems to be just picking date if it gets passed in a null value - it might be the same as calling `carbon::now()` or something like that.

Do you prefer `!empty()` then? should I change the one `array_key_exists()` to use `!empty()`?

Reply:
Yeah lets change the all to empty I think